### ISSUE TITLE
No such module 'CircleSlider' - Unable to run demo

### ISSUE THREAD

**Issue by alexgpark** [shushutochako/CircleSlider#7]
1) Cloned repo - no problem
2) Ran pod install successfully - no problem
3) Opened up the Project file and ran - Build Failed > "No Such Module 'CircleSlider'" for line 10 "import CircleSlider" in ViewController.swift
![screen shot 2016-03-12 at 10 02 34 pm](https://cloud.githubusercontent.com/assets/6458377/13735545/36f780f4-e968-11e5-832f-9f4e0f4b46e0.png)
<img width="266" alt="screen shot 2016-03-13 at 10 09 21 pm" src="https://cloud.githubusercontent.com/assets/6458377/13735552/457b81e8-e968-11e5-9c1e-56e5475f7cfe.png">

**Comment by nahouto** [comment]
you have to do pod install before building the demo

**Comment by alexgpark** [comment]
Yes, if you look up at #2, I did that successfully but still no luck at
Step #3
- Alex

**Comment by shushutochako** [comment]
Hi @alexgpark .
Do you opening the CircleSlider.xcworkspace?
When you run, you must open a CircleSlider.xcworkspace instead of CircleSlider.xcodeproj.

**Comment by alexgpark** [comment]
Yes, I actually tried both the proj and workspace, and neither of them
worked initially.

However, I just figured out the issue. I needed to do this:
http://i.imgur.com/JYFf8LK.gifv
- Alex

On Mon, Mar 14, 2016 at 6:43 PM, Nobuyasu Masujima <notifications@github.com

**Comment by shushutochako** [comment]
Hi @alexgpark 
I apologize for the late reply. At present, The cause is unclear. 
This issue is not yet resolved?
